### PR TITLE
fixes #72

### LINKED ISSUE #72: Add Solution for Valid Anagram Problem
Add solution for Valid Anagram Problem(Problem number 242)
Link to the problem:- https://leetcode.com/problems/valid-anagram/

### PR DESCRIPTION
**Pull request by Anonymous5164**
fixes #72 
It contains solution for Valid anagram problem

### CODE REVIEW

**Review by md-shamim-ahmad — CHANGES_REQUESTED** [review]
Hi
Thanks for your contribution.
Please change something in your code.
Rename the file ```Valid Anagram.cpp``` to ```242. Valid Anagram.cpp``` and format the code in ```clang-format```

**Comment by Anonymous5164** [comment]
Renamed the file to "242. Valid Aanagram.cpp" in "clang-format" as directed, kindly review the same and let me know if any further changes are needed.

File: LeetCode Problems/201 to 300/242. Valid Anagram/242. Valid Anagram.cpp [hunk 1]
```diff
@@ -0,0 +1,28 @@
+class Solution {
+public:
+    bool isAnagram(string s, string t) {
+
+if(s.size()!= t.size()){
+    return false;
+}
+
+        int alphabets[26]= {0};
```
Comment (md-shamim-ahmad) [inline LeetCode Problems/201 to 300/242. Valid Anagram/242. Valid Anagram.cpp]: Here
```cpp
int alphabets[26]= {0};
```
will be as 
```cpp
int alphabets[26] = {0};
```

File: LeetCode Problems/201 to 300/242. Valid Anagram/242. Valid Anagram.cpp [hunk 2]
```diff
@@ -0,0 +1,28 @@
+class Solution {
+public:
+    bool isAnagram(string s, string t) {
+
+if(s.size()!= t.size()){
```
Comment (md-shamim-ahmad) [inline LeetCode Problems/201 to 300/242. Valid Anagram/242. Valid Anagram.cpp]: The format of your code not good.
Like these line,
```cpp
    bool isAnagram(string s, string t) {

if(s.size()!= t.size()){
    return false;
}
```
correct format will be: 
```cpp
    bool isAnagram(string s, string t) {

    if (s.size() != t.size()) {
        return false;
    }
```

**Review by md-shamim-ahmad — CHANGES_REQUESTED** [review]
So format this code in ```Clang-Format``` style.

File: LeetCode Problems/201 to 300/242. Valid Anagram/242. Valid Anagram.cpp (hunk 2 above)
Comment (Anonymous5164) [reply LeetCode Problems/201 to 300/242. Valid Anagram/242. Valid Anagram.cpp]: Sorry my bad, I solved it on Leetcode and pasted the code from leetcode's IDE, I have reformatted and added indentations to make my code consistent and easy to read, kindly review the same and let me know if any further changes are required

**Review by md-shamim-ahmad — APPROVED** [review]

**Pull request merged**